apps/admin-x-activitypub/src/components/feed/FeedItem.tsx

```diff
@@ -302,7 +302,7 @@ const FeedItem: React.FC<FeedItemProps> = ({actor, object, layout, type, comment
         return (
             <>
                 {object && (
-                    <div className={`group/article relative cursor-pointer pt-6`} onClick={onClick}>
+                    <div className={`group/article relative cursor-pointer pt-6`} data-uuid={object.id} onClick={onClick}>
```

Comment:
Just an observation - `object.id` is a `URL` and not a `UUID`. To avoid ambiguity we could call this `data-object-id` instead? or am I being too pedantic 😅 @allouis @djordjevlais

Reply:
Have changed this to `data-object-id` 👍

Reply:
😁 Sure, we can call it that! I didn't wanna block on it tho